seirmo/simulation.py

```diff
@@ -0,0 +1,36 @@
+#
+# This file is part of seirmo (https://github.com/SABS-R3-Epidemiology/seirmo/)
+# which is released under the BSD 3-clause license. See accompanying LICENSE.md
+# for copyright notice and full license details.
+#
+
+import numpy as np
+
+import seirmo as se
+
+
+class SimulationController(object):
+    """SimulationController Class:
+
+    Runs the simulation of any model and controls outputs
+
+    Parameters
+    ----------
+    model: seirmo.ForwardModel class
+    start: simulation start time
+    end: simulation end time
+    """
+
+    def __init__(self, model, start, end, n_times): # noqa
```

Comment:
could we remove the `n_times` argument here. 

The SimulatinoController will be mostly there for interfacing with the GUI later, so I was thinking that it makes more sense to have a method that allows you to adapt the number of points for evaluation (i.e. the resolution).